Apps/System/systemapp.c

```diff
@@ -120,12 +120,10 @@ static void exit_to_watchface(struct Menu *menu, void *context)
 static void systemapp_window_load(Window *window)
 {
     Layer *window_layer = window_get_root_layer(s_main_window);
-#ifdef PBL_RECT
-    s_menu = menu_create(GRect(0, 16, DISPLAY_COLS, DISPLAY_ROWS - 16));
-#else
-    // Let the menu draw behind the statusbar so it is perfectly centered
-    s_menu = menu_create(GRect(0, 0, DISPLAY_COLS, DISPLAY_ROWS));
-#endif
+
+    // For round, let the menu draw behind the statusbar so it is perfectly centered
+    s_menu = menu_create(PBL_IF_RECT_ELSE(GRect(0, 16, DISPLAY_COLS, DISPLAY_ROWS - 16), GRect(0, 0, DISPLAY_COLS, DISPLAY_ROWS)));
```

Comment:
Maybe on two different lines, aligned with spaces, to better visually indicate the difference?  i.e., 

```c
    s_menu = menu_create(PBL_IF_RECT_ELSE(GRect(0, 16, DISPLAY_COLS, DISPLAY_ROWS - 16), // rect
                                          GRect(0, 0, DISPLAY_COLS, DISPLAY_ROWS)        // round
                                         ));
```